lib/templates/components/content/CeDynamic.js

```diff
@@ -6,16 +6,21 @@ export default {
      */
     data: {
       type: [Object, Array],
-      required: true,
```

Comment:
Why are you removing the „required” flag?

Reply:
Good point, but in this case we should add required and remove default because it make no sense here. Additionally we should change type to Object, because in the render we use this `data` with object keys - I have missed it.  @gdulamat Could you change this please ?